src/stdlib/mezmo_redact.rs

```diff
@@ -935,7 +944,7 @@ mod test {
             args: func_args![
                 value: value!({
                     "arr": [
-                        "contains 180011111111122, a JCB credit card",
+                        "contains 1800222233334444, a JCB credit card",
```

Comment:
in the filters list below, might be worth moving the `phone_number` entry to the front of the array, or at least earlier than the `us_social_security_number` one? just to make the `sort_by()` really work hard

Reply:
Done